chsdi/views/features.py

```diff
@@ -35,6 +36,10 @@
 
 
 MAX_FEATURES = 201
+CACHE_TTL = 60 * 30  # 30 minutes
+CACHE_SIZE = 100  # Number of layers metadata (to multiplied by lang)
```

Comment:
I like comments, they help us understand the code. 
I would modify the comment for cache size to 
"# Number of layers metadata for each language. "